js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -625,6 +625,17 @@ const DropDownEditor = TextBox.inherit({
     },
 
     _dimensionChanged: function() {
+        // This part hides the popup on window resize if an editor was hidden (T1133813)
+        // Use the resize observer to close the popup after hiding the editor instead
```

Comment:
```suggestion
        // TODO: Use ResizeObserver to hide popup after editor visibility change
        // instead of window's dimension change,
```

Reply:
https://github.com/DevExpress/DevExtreme/pull/23483/commits/709cfa030c245fe825d90854ffcbc9655bae3ca2